specification/electronic-prescription-service-api.yaml

```diff
@@ -133,8 +133,48 @@ info:
     We are hoping to make the onboarding process as lightweight and as self-service as possible.
 
     More details to follow.
-
-      
+    
+    ## Responses
+    ### 2xx responses
+    **Note: This section is under development and subject to change. All Spine responses are returned with 200 HTTP codes until a Spine translation service is built. This does not affect the sandbox environment.**
+    | Endpoint              | HTTP code | Response format                                                                                                                                             |
+    | --------------------- | --------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------- |
+    | POST /$prepare        | 200       | [See /$prepare response body](https://digital.nhs.uk/developer/api-catalogue/electronic-prescription-service-fhir#api-Prescribing-preparePrescription)      |
+    | POST /$convert        | 200       | [See /$convert response body](https://digital.nhs.uk/developer/api-catalogue/electronic-prescription-service-fhir#api-Prescribing-convertPrescription)      |
+    | POST /$process-message| 200       | [See /$process-message response body](https://digital.nhs.uk/developer/api-catalogue/electronic-prescription-service-fhir#api-Prescribing-sendPrescription) |
+    | POST /$prepare        | 200       | [See /$prepare response body](https://digital.nhs.uk/developer/api-catalogue/electronic-prescription-service-fhir#api-Prescribing-preparePrescription)      |
+    | POST /$convert        | 200       | [See /$convert response body](https://digital.nhs.uk/developer/api-catalogue/electronic-prescription-service-fhir#api-Prescribing-convertPrescription)      |
+    | POST /$process-message| 200       | [See /$process-message response body](https://digital.nhs.uk/developer/api-catalogue/electronic-prescription-service-fhir#api-Prescribing-sendPrescription) |
+    ### 4xx responses
+    **Note: This section is under development and subject to change. All Spine responses are returned with 200 HTTP codes until a Spine translation service is built. This does not affect the sandbox environment.**
+    | Endpoint              | HTTP code | Response format                           | Environment      | Cause                 |
+    | --------------------- | --------- | ----------------------------------------- | ---------------- | --------------------- |
+    | Any                   | 400       | See "Unsupported operation" example below | Sandbox          | Unsupported operation |
```

Comment:
not sure if theyre visually different to a user but we have other types of 400 responses i think? in `src/server/routes/payload.ts` we return 4xx - notFound ad also 4xx - badRequest, maybe these are different? not sure lol

Reply:
hmm thats the signing repo tho so is it relevant here?

Reply:
wow yea okay im an idiot lol